readme.md

```diff
@@ -302,23 +312,134 @@ Sets `options.method` to the method name and makes a request.
 
 #### Instances
 
-#### got.create([options])
+#### got.extend([options])
 
-Configure a new `got` instance with default `options`:
+Configure a new `got` instance with default `options` and custom `endpoint` (optional):
 
 ```js
 (async () => {
-	const client = got.create({headers: {'x-foo': 'bar'}});
-	const {headers} = await client.get('httpbin.org/headers', {json: true}).body;
+	const client = got.extend({headers: {'x-foo': 'bar'}, endpoint: 'httpbin.org/'});
+	const {headers} = (await client.get('headers', {json: true})).body;
 	//=> headers['x-foo'] === 'bar'
 
-	const jsonClient = client.create({json: true, headers: {'x-baz': 'qux'}});
-	const {headers: headers2} = await jsonClient.get('httpbin.org/headers').body;
+	const jsonClient = client.extend({json: true, headers: {'x-baz': 'qux'}});
+	const {headers: headers2} = (await jsonClient.get('headers')).body;
 	//=> headers2['x-foo'] === 'bar'
 	//=> headers2['x-baz'] === 'qux'
 })();
 ```
 
+#### got.create(settings)
+
+Example: [gh-got](https://github.com/sindresorhus/gh-got/blob/master/index.js)
+
+Configure a new `got` instance with provided settings:
+
+##### [options](#options)
+
+To inherit from parent, set it as `got.defaults.options` or use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).
+
+##### methods
+
+Type: `object`
+
+Array of supported request methods.
+
+To inherit from parent, set it as `got.defaults.methods`.
+
+##### handler
+
+Type: `function`<br>
+Default: `undefined`
+
+Function making additional changes to the request.
+
+To inherit from parent, set it as `got.defaults.handler`.<br>
+To use the default handler, set it as `null` or `undefined`.
```

Comment:
Couldn't this just be: `To use the default handler, just omit specifying this.`?